pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -458,6 +458,7 @@ texlinks = stdenv.mkDerivation rec {
   '';
 };
 
+inherit asymptote;
```

Comment:
As far as I can tell, building `asymptote` does not really require texlive -- it's just used to generate its documentation. As the documentation is also available in the texlive-doc-package and using texdoc is far more convenient than manually browsing to the pdf in the `asymptote`-package anyway, just not generating the documentation in the `asymptote` derivation would be by far the best option to break up this inconvenient dependency chain.

Unfortunately, asymptote's build system is terrible and “not generating the documentation” does not appear to be a usecase they support, so disabling that is slightly painful.

I'd rather leave asympote in, the rebuilds only annoy people working on texlive (i.e. you). But as nobody has ever complained about `asymptote` not being in texlive in nixpkgs, removing it again is also fine.

btw: xasy is also broken.